src/popup/router/pages/Transactions.vue

```diff
@@ -104,17 +105,13 @@ export default {
         page: 1,
         recent: true,
       });
-      const diff = differenceWith(transactions, this.transactions, isEqual);
-      this.updateTransactions({ latest: true, transactions: diff });
+      this.updateTransactions({ latest: true, transactions });
     },
     updateTransactions({ transactions, latest }) {
```

Comment:
I would refactor this method futher. Please check the changes in 59aac0eb0a5f017f4552c7f6d092dc7cc6f0409e and apply if they are acceptable